app/components/app/app.js

```diff
@@ -367,7 +367,7 @@ let mapDispatchToProps = dispatch => bindActionCreators({
 
 let mergeProps = (stateProps, dispatchProps, ownProps) => {
   var api = ownProps.routes[0].api;
-  return _.merge({}, ownProps, stateProps, dispatchProps, {
+  return Object.assign({}, ownProps, stateProps, dispatchProps, {
```

Comment:
All uses of _.merge in mergeProps() functions used when connecting redux state to components has been removed.